src/views/CharacterListView.js

```diff
@@ -2,34 +2,45 @@ import React from "react";
 import { connect } from "react-redux";
 
 import { CharacterList } from "../components";
+
 // import actions
+import { fetchingPeople } from "../actions/";
 
 class CharacterListView extends React.Component {
-  constructor() {
-    super();
-  }
+    constructor(props) {
+        super();
+    }
+
+    componentDidMount() {
+        this.props.fetchingPeople();
 
-  componentDidMount() {
-    // call our action
-  }
+        // call our action
+    }
 
-  render() {
-    if (this.props.fetching) {
-      // return something here to indicate that you are fetching data
+    render() {
+        if (this.props.fetching) {
+            // return something here to indicate that you are fetching data
+        }
+        return (
+            <div className="CharactersList_wrapper">
+                <CharacterList characters={this.props.characters} />
+            </div>
+        );
     }
-    return (
-      <div className="CharactersList_wrapper">
-        <CharacterList characters={this.props.characters} />
-      </div>
-    );
-  }
 }
 
 // our mapStateToProps needs to have two properties inherited from state
 // the characters and the fetching boolean
+const mapStateToProps = state => {
+    return {
+        characters: state.charsReducer.characters,
+        fetching: state.charsReducer.fetching
```

Comment:
the one thing I forgot with Francis - ugh I can't believe I forgot that.

Great work here!